tests/checks/events.py

```diff
@@ -0,0 +1,112 @@
+from typing import Dict, List, Optional
```

Comment:
When you will create tests for each event function this file will be not needed anymore. Instead of doing asserts like in `was_digital_good_download_event_properly_generated` you can easily use mock and `assert with given arguments`

Reply:
I kept the file, renamed it to `customer`, it checks the consistency between the API and the dashboard as we have two totally different logic for deleting customers. They don't process the same way, so they can't call a wrapped logic.

Only thing we can do, is mock them and assert the call.